[2022-03-31 13:12:39,709] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: postgres_dag.get_data_uba scheduled__2022-03-30T18:11:47.725926+00:00 [queued]>
[2022-03-31 13:12:39,716] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: postgres_dag.get_data_uba scheduled__2022-03-30T18:11:47.725926+00:00 [queued]>
[2022-03-31 13:12:39,716] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-03-31 13:12:39,717] {taskinstance.py:1244} INFO - Starting attempt 1 of 1
[2022-03-31 13:12:39,717] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-03-31 13:12:39,931] {taskinstance.py:1264} INFO - Executing <Task(_PythonDecoratedOperator): get_data_uba> on 2022-03-30 18:11:47.725926+00:00
[2022-03-31 13:12:39,934] {standard_task_runner.py:52} INFO - Started process 442 to run task
[2022-03-31 13:12:39,938] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'postgres_dag', 'get_data_uba', 'scheduled__2022-03-30T18:11:47.725926+00:00', '--job-id', '212', '--raw', '--subdir', 'DAGS_FOLDER/postgres_conn.py', '--cfg-path', '/tmp/tmpunoo_6d8', '--error-file', '/tmp/tmprsag2clh']
[2022-03-31 13:12:39,938] {standard_task_runner.py:77} INFO - Job 212: Subtask get_data_uba
[2022-03-31 13:12:40,111] {logging_mixin.py:109} INFO - Running <TaskInstance: postgres_dag.get_data_uba scheduled__2022-03-30T18:11:47.725926+00:00 [running]> on host LAPTOP-5HJG55H7.localdomain
[2022-03-31 13:12:40,443] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=postgres_dag
AIRFLOW_CTX_TASK_ID=get_data_uba
AIRFLOW_CTX_EXECUTION_DATE=2022-03-30T18:11:47.725926+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-30T18:11:47.725926+00:00
[2022-03-31 13:12:40,449] {base.py:70} INFO - Using connection to: id: airflow-universities. Host: training-main.cghe7e6sfljt.us-east-1.rds.amazonaws.com, Port: 5432, Schema: training, Login: alkymer, Password: ***, extra: {}
[2022-03-31 13:12:41,124] {python.py:175} INFO - Done. Returned value was: None
[2022-03-31 13:12:41,131] {taskinstance.py:1272} INFO - Marking task as SUCCESS. dag_id=postgres_dag, task_id=get_data_uba, execution_date=20220330T181147, start_date=20220331T181239, end_date=20220331T181241
[2022-03-31 13:12:41,480] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-03-31 13:12:41,497] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
